tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php

```diff
@@ -453,7 +456,18 @@ private function prepareExternalBucketForRegistration(string $description): List
         $iamExchangerPolicy->setBindings($binding);
         $analyticHubClient->setIamPolicy($dataExchange->getName(), $iamExchangerPolicy);
 
-        return $createdListing;
+        $pattern = '/projects\/([\w-]+)\/locations\/([\w-]+)\/dataExchanges\/([\w-]+)\/listings\/([\w-]+)/';
```

Comment:
Vzhledem k tomu, že pro regiatraci stejně máme toho klienta - tak jestli to jde, tak by bylo fajn to parsovat v něm... pokud nemá jen skládání

Reply:
Hm client posiela len $path array, ale prepisal som toto a pouzivam parsovanie priamop analytic hub service